.github/workflows/ci.yml

```diff
@@ -26,7 +26,54 @@ concurrency:
   cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
 
 jobs:
-  distributed_tripolar:
+  sharded_tripolar:
+    name: Sharding Tests - Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 120
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.10'
+        os:
+          - ubuntu-latest
+        arch:
+          - x64
+    steps:
+      - name: Set XLA runtime
+        run: |
+          cat <<EOF > LocalPreferences.toml
+          [Reactant]
+          xla_runtime = "IFRT"
+          EOF
+      - name: Instantiate environment
+        timeout-minutes: 90
+        shell: julia --color=yes --project {0}
+        run: |
+          using Pkg
+          Pkg.Registry.update()
+          Pkg.add([
+              PackageSpec(; name="Reactant", rev="ap/sdy_fixes2"),
+          #     PackageSpec(; name="ReactantCore", rev="main"),
+          #     PackageSpec(; name="Oceananigans", rev="main"),
+          #     PackageSpec(; name="ClimaOcean", rev="main"),
+          ])
+          Pkg.instantiate()
+      - name: Run sharding tests
+        - uses: actions/checkout@v4
+        - uses: julia-actions/setup-julia@v2
+          with:
+            version: ${{ matrix.version }}
+            arch: ${{ matrix.arch }}
+        - uses: julia-actions/cache@v2
+        - uses: julia-actions/julia-buildpkg@v1
+        - uses: julia-actions/julia-runtest@v1
+          env:
+            REACTANT_TEST: true
+            XLA_FLAGS: "--xla_force_host_platform_device_count=4"
+            TEST_GROUP: "sharding"
+
+  mpi_tripolar:
```

Comment:
@giordano halp

Reply:
what specifically?

Reply:
https://github.com/CliMA/Oceananigans.jl/actions/runs/14047271790/workflow syntax issue

Reply:
how do you debug these things?

Reply:
oh i see there is a message